open_academy/models/open_academy_session.py

```diff
@@ -8,5 +10,54 @@ class Session(models.Model):
     name = fields.Char(required=True)
     description = fields.Text()
     start_date = fields.Datetime(default=fields.Date.context_today)
+    end_date = fields.Datetime(compute='_compute_end_date')
+    active = fields.Boolean(default=True)
     duration = fields.Integer()
-    number_of_seats = fields.Integer()
+    number_of_seats = fields.Integer(default=1)
+    instructor_id = fields.Many2one('res.partner', string='Instructor')
```

Comment:
String is not need it when it's the same that field.
```suggestion
    instructor_id = fields.Many2one('res.partner')
```